src/parsers/pm2ps_parser.jl

```diff
@@ -374,7 +374,7 @@ function make_line(name, d, bus_f, bus_t)
         x=d["br_x"],
         b=(from=d["b_fr"], to=d["b_to"]),
         rate=d["rate_a"],
-        anglelimits=(min=rad2deg(d["angmin"]), max=rad2deg(d["angmax"])),
+        anglelimits=(min=deg2rad(d["angmin"]), max=deg2rad(d["angmax"])),
```

Comment:
@claytonpbarrows @jd-lara Angle limit violations in the CDM parser are now fixed, and now errors are popping up in pm2ps parser. 

Dan and I fixed the line angle limit violations by adding in deg2rad() in make_line(), but should we convert the angle in make_bus() as well? The test data values for the bus angles are 0, so no error is being thrown.

Reply:
Adding the conversion is necessary, both MatPower and PowerModels assume angles in degrees. 

Also, @KyraDiF take a look here for what changes are being introduced in the matpower parser. 

https://github.com/NREL/PowerSystems.jl/blob/570c5d08a20241216a3789a76f209b1d9520ce93/src/parsers/pm_io/common.jl#L49

Reply:
@jd-lara 
Now that I've added these angle conversions, I'm getting another error in the CDM parsing tests. 
These tests are failing:
@test cdm_branches[2].rate == get_branch(mpsys, cdm_branches[2]).rate
@test cdm_branches[6].rate == get_branch(mpsys, cdm_branches[6]).rate

It looks like the matpower parsing is reading in the angles wrong, or in a way I don't understand. In RTS_GMLC file, the angles limits are all -180 to 180, when the data is parsed, the angle limits are read in as -1.0472 and 1.0472:

Try this: 
```
data = parse_file("data/matpower/RTS_GMLC.m")
data["branch"]["49"] 
```

Reply:
Have you checked the changes introduced by PowerModels checking code? It looks to me that it could be that.

Reply:
yes